src/sentry/issues/grouptype.py

```diff
@@ -391,7 +391,7 @@ class PerformanceNPlusOneAPICallsGroupType(GroupType):
 
 
 @dataclass(frozen=True)
-class PerformanceNPlusOneAPICallsExperimentalGroupType(GroupType):
+class PerformanceNPlusOneAPICallsExperimentalGroupType(PerformanceGroupTypeDefaults, GroupType):
```

Comment:
Adding the default Noise Config for the experiment -> Only create an issue when more than 3 are seen in 24 hours